### ISSUE TITLE
Layout broken when book has style on `html`/`body` with `!important`

### ISSUE THREAD

**Issue by afduggirala** [johnfactotum/foliate#1122]
Hello,
I just installed Foliate 3.0.0 from Flatpak on Fedora 38.
Now I am seeing again the issue reported here: https://github.com/johnfactotum/foliate/issues/840

I believe this behavior had stopped happening at some point in the latest releases. Now in this new version I am seeing it again,

thanks,

**Comment by johnfactotum** [comment]
- Are you using HiDPI? Fractional scaling?
- Are you zooming in our out?
- Does it happen with every book?
- Does it happen if you use this online viewer, in any browser? https://johnfactotum.github.io/foliate-js/reader.html
- How does it behave when you resize the window?

**Comment by afduggirala** [comment]
- It's only happening with one book, I tried 5 other books that work correctly. I tried redownloading the book from the site but that didn't work. The book can be downloaded from here. https://git-scm.com/book/en/v2  In epub format. (I want to be a git pro)
- My laptop has a high-dpi screen, I am working on an external monitor right now though, so Im not sure how that would have an effect.
- I am not zooming in or out.
- Yes. It also happens when using the online viewer.
- Resizing the windows doesn't have any real effect. It is still not centered, it just changes the amount of text in each "column".
